### ISSUE TITLE
pushbullet update version 18 to use fcm

### ISSUE THREAD

**Issue by patrickdrd** [microg/android_packages_apps_GmsCore#703]
"We just updated to Firebase Cloud Messaging now that Google Cloud Messaging (GCM) is going away"

so I guess we can't update

please @mavrin, we need an update with fcm

**Comment by Mavrin** [comment]
@patrickdrd  Look like I was mentioned by mistake. @mar-v-in

**Comment by Strel0ka** [comment]
[https://github.com/microg/android_packages_apps_GmsCore/issues/693#issuecomment-462394414](Regarding the work of marvin)

**Comment by patrickdrd** [comment]
yes, I know, if you see one post above, you'll see that I called him :)
